tfe/provider.go

```diff
@@ -106,25 +106,25 @@ func Provider() *schema.Provider {
 		},
 
 		DataSourcesMap: map[string]*schema.Resource{
-			"tfe_organizations":           dataSourceTFEOrganizations(),
-			"tfe_organization":            dataSourceTFEOrganization(),
-			"tfe_agent_pool":              dataSourceTFEAgentPool(),
-			"tfe_ip_ranges":               dataSourceTFEIPRanges(),
-			"tfe_oauth_client":            dataSourceTFEOAuthClient(),
-			"tfe_organization_membership": dataSourceTFEOrganizationMembership(),
-			"tfe_organization_run_task":   dataSourceTFEOrganizationRunTask(),
-			"tfe_slug":                    dataSourceTFESlug(),
-			"tfe_ssh_key":                 dataSourceTFESSHKey(),
-			"tfe_team":                    dataSourceTFETeam(),
-			"tfe_team_access":             dataSourceTFETeamAccess(),
-			"tfe_team_project_access":     dataSourceTFETeamProjectAccess(),
-			"tfe_workspace":               dataSourceTFEWorkspace(),
-			"tfe_workspace_ids":           dataSourceTFEWorkspaceIDs(),
-			"tfe_workspace_run_task":      dataSourceTFEWorkspaceRunTask(),
-			"tfe_variables":               dataSourceTFEWorkspaceVariables(),
-			"tfe_variable_set":            dataSourceTFEVariableSet(),
-			"tfe_policy_set":              dataSourceTFEPolicySet(),
-			"tfe_organization_members":    dataSourceTFEOrganizationMembers(),
+			"tfe_organizations":               dataSourceTFEOrganizations(),
+			"tfe_organization":                dataSourceTFEOrganization(),
+			"tfe_agent_pool":                  dataSourceTFEAgentPool(),
+			"tfe_ip_ranges":                   dataSourceTFEIPRanges(),
+			"tfe_oauth_client":                dataSourceTFEOAuthClient(),
+			"tfe_organization_membership":     dataSourceTFEOrganizationMembership(),
+			"tfe_organization_run_task":       dataSourceTFEOrganizationRunTask(),
+			"tfe_organization_workspace_tags": dataSourceTFEOrganizationTags(),
```

Comment:
```suggestion
			"tfe_organization_tags": dataSourceTFEOrganizationTags(),
```

One more and I think we are good